build.gradle

```diff
@@ -375,6 +375,7 @@ licenseReport {
       'io.netty:netty-tcnative-classes',
       'org.jetbrains.kotlinx:kotlinx-coroutines-bom',
       'org.jetbrains.kotlinx:kotlinx-coroutines-core',
+      'org.jetbrains.kotlin:kotlin-stdlib-common',
```

Comment:
updated library does not have readable license anymore (like some other org.jetbrains deps).